src/VertexArrayObject.js

```diff
@@ -92,6 +92,12 @@ VertexArrayObject.prototype.bind = function()
         {
             this.dirty = false;
             this.activate();
+        } else
```

Comment:
@ivanpopelyshev could you return before `else`? Linting is not as strict on this repo.